federation/cmd/federation-controller-manager/app/options/options.go

```diff
@@ -115,5 +118,6 @@ func (s *CMServer) AddFlags(fs *pflag.FlagSet) {
 	fs.IntVar(&s.APIServerBurst, "federated-api-burst", s.APIServerBurst, "Burst to use while talking with federation apiserver")
 	fs.StringVar(&s.DnsProvider, "dns-provider", s.DnsProvider, "DNS provider. Valid values are: "+fmt.Sprintf("%q", dnsprovider.RegisteredDnsProviders()))
 	fs.StringVar(&s.DnsConfigFile, "dns-provider-config", s.DnsConfigFile, "Path to config file for configuring DNS provider.")
+	fs.BoolVar(&s.EnableIngressController, "enable-ingress-controller", s.EnableIngressController, "Enable federation ingress controller.")
```

Comment:
How about adding the `runtime-config` option (which is available to api server) to controller manager as well for enable/disable specific resource?